src/couch/test/eunit/couchdb_views_tests.erl

```diff
@@ -474,6 +475,7 @@ can_update_views_with_old_collators({_, {DbName, Files}}) ->
 view_collator_auto_upgrade_on_open({_, {DbName, Files}}) ->
     ?_test(begin
         [_NewDbFilePath, ViewFilePath, ViewFilePath] = Files,
+        ok = config:set("query_server_config", "commit_freq", "0", false),
```

Comment:
Should this setting be deleted after the test has run?

Reply:
For this particular set of tests technically not as important as it's set in memory only (doesn't get written) and on every test we restart all the couch apps including couch config since they run in a `foreach` test group https://github.com/apache/couchdb/blob/4bcbf1fd649f6e556fab069487aa96fad1c031e9/src/couch/test/eunit/couchdb_views_tests.erl#L250-L258